master/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml

```diff
@@ -12,9 +12,9 @@ data:
   # defined below.
   etcd_endpoints: "http://10.96.232.136:6666"
 
-  # True enables BGP networking, false tells Calico to enforce
+  # Disable Calico networking backend, which tells Calico to enforce
   # policy only, using native networking.
-  enable_bgp: "true"
+  calico_backend: "none"
```

Comment:
Yeah, that's exactly what the reason was.

The goal was to be able to configure Calico completely through this config map.  That said, it doesn't buy us much - we've still got three different manifests as it is today.

I'd be OK with dropping this configuration field from the ConfigMap, provided it defaults to BIRD.

The k8s datastore driver is the only one that needs this field set to "none".